### DISCUSSION TITLE
how to display image?

### DISCUSSION THREAD

**Discussion by tigercoding56** [InfiniTimeOrg/InfiniTime#1680] [Q&A] [answered]
i want to display a 185x150 image as background for the watchface , how would i do this ?

**Comment by JF002** [comment, marked as answer]
You can have a look at [this commit](https://github.com/InfiniTimeOrg/InfiniTime/commit/10beda289bf0ccae18c89872e8da48ee44be14ca) which is part of the [InfiniTime 1.11 FOSDEM 2023 edition](https://github.com/InfiniTimeOrg/InfiniTime/releases/tag/1.11.0-fosdem-edition), where we added the picture of the FOSDEM logo to the background of the digital and analog watchface.
